ci-operator/step-registry/code-ready/snc/e2e/test/code-ready-snc-e2e-test-commands.sh

```diff
@@ -78,6 +78,12 @@ LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute scp \
   --zone "${GOOGLE_COMPUTE_ZONE}" \
   --recurse /opt/snc packer@"${INSTANCE_PREFIX}":~/snc
 
+# xmlstarlet package is part of epel so we need to enable the epel repo
+LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute --project "${GOOGLE_PROJECT_ID}" ssh \
+  --zone "${GOOGLE_COMPUTE_ZONE}" \
+  packer@"${INSTANCE_PREFIX}" \
+  --command "sudo yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm"
```

Comment:
actually it might be better to do this as part of the snc script?

Reply:
@cfergeau snc script can be run on fedora where this package already available

Reply:
We could check `/etc/os-release` and adjust the packages depending on this.

Reply:
@cfergeau yes we can then you also need to parse which version of centos or rhel to make sure you get correct epel repo out of it, if you add all those logic in snc then go ahead otherwise I also think we should need some alternative which should available in default repo so we don't have to do all these.

Reply:
I was initially wondering whether to use xmlstarlet or a custom static go binary for this, maybe the go option would have been better... We could also use a few lines of python-lxml https://lxml.de/xpathxslt.html

Reply:
Actually xmlstarlet use can be removed, and replaced with `virsh capabilities | xmllint --xpath '/capabilities/guest/arch[@name="x86_64"]/domain[@type="kvm"]'  -` so this change should not be needed either.

Reply:
Not a preferred solution... another dependency, and another possible upgrde issue (as this relies on an external repo).